src/components/survey/__tests__/sorting-hat-reducer.test.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * @jest-environment jsdom
+ */
+
+// could eventually test that cioIdentify and track are being called appropriately
+
+import {cioIdentify} from '../cio-identify'
+import {sortingHatReducer, sortingHatInitialState} from '../sorting-hat-reducer'
+import {track} from 'utils/analytics'
+
+jest.mock('../cio-identify', () => ({
+  cioIdentify: jest.fn(() => null),
+}))
+
+jest.mock('utils/analytics', () => ({
+  track: jest.fn(),
+}))
+
+const defaultFirstQuestion =
+  sortingHatInitialState.data[sortingHatInitialState.currentQuestionKey]
+
+describe('empty action', () => {
+  test('should return the initial state', () => {
+    expect(sortingHatReducer(sortingHatInitialState, {})).toStrictEqual(
+      sortingHatInitialState,
+    )
+  })
+})
+
+describe('load action', () => {
+  test('should close sorting hat for load action with undefined subscriber after finished loading', () => {
+    expect(
+      sortingHatReducer(sortingHatInitialState, {
+        type: 'load',
+        loadingSubscriber: 'false',
+      }),
+    ).toStrictEqual({
+      ...sortingHatInitialState,
+      question: defaultFirstQuestion,
+      closed: true,
```

Comment:
and then here we are expecting it to strictly equal some object that gets formed inline.

What is this? Giving it a name and assigning it to a `const` would bring a lot of clarity here.